Add MSI installation to documentation

russcam opened the pull request:
Move installation documentation for Windows with the .zip archive into the zip and tar installation documentation, and clearly indicate any differences for installing on macOS/Linux and Windows.

clintongormley commented on docs/reference/setup/install/windows.asciidoc:
```diff
@@ -25,236 +28,408 @@ endif::[]
 
 [horizontal]
-`install`:: Install Elasticsearch as a service
+`INSTALLDIR`:: 
```
This seems to be a repetition of a section above.  Could these two not be combined?

russcam replied:
This section serves as the canonical set of properties that the installer accepts (case-sensitive) so I'll look to remove the repeated information above

russcam commented:
Thanks for taking a look @clintongormley!

> One thought - combining the windows zip instructions with zip-targz makes the page rather long and complicated. Might be easier to keep a separate zip-on-windows page?

Was thinking the same too, the waters are a little bit muddied in the single file 😄 I'll update to separate

russcam commented:
I've updated @clintongormley, if you wouldn't mind taking another look. I've split out the windows zip installation instructions into a separate page

clintongormley commented:
thanks @russcam

russcam commented:
@clintongormley should this be ported to `5.x` and `5.5`?
